system/settings/win32.xml

```diff
@@ -29,13 +20,6 @@
     </category>
   </section>
   <section id="system">
-    <category id="videoscreen">
-      <group id="1">
-        <setting id="videoscreen.fakefullscreen">
```

Comment:
This can already go for osx. It only works for 10.6 and is hidden for later osx versions - as we have now 10.7 as minimum it can go...